ansible/roles/datadog/templates/http.yaml.j2

```diff
@@ -0,0 +1,12 @@
+init_config:
+
+instances:
+  - name: {{ env_monitoring_id }}
+    url: https://{{ SITE_HOST }}/serverup.txt
+    disable_ssl_validation: false
+    check_certificate_expiration: true
+    days_warning: 28
+    days_critical: 14
```

Comment:
I probably don't know what this means, but this seems like a lot of days?

Reply:
this is how much many days before the cert expires that the alert will go from `warning` to `critical`